src/main/java/io/github/thebusybiscuit/mobcapturer/mobs/AbstractHorseAdapter.java

```diff
@@ -23,7 +22,7 @@ public AbstractHorseAdapter(Class<T> entityClass) {
     public List<String> getLore(JsonObject json) {
         List<String> lore = super.getLore(json);
 
-        lore.add(ChatColor.GRAY + "Jump Strength: " + ChatColor.WHITE + DoubleHandler.fixDouble(json.get("jumpStrength").getAsDouble()));
+        lore.add(ChatColor.GRAY + "Jump Strength: " + ChatColor.WHITE + json.get("jumpStrength").getAsDouble());
```

Comment:
The value should be sanitized.
So please revert it back (well, use the new method in this case since the old one was removed).

```suggestion
        lore.add(ChatColor.GRAY + "Jump Strength: " + ChatColor.WHITE + NumberUtils.roundDecimalNumber(json.get("jumpStrength").getAsDouble()));
```